pkg/install/install.go

```diff
@@ -156,6 +163,7 @@ func (i *Installer) Install(ctx context.Context, installConfig *installconfig.In
 		},
 		api.InstallPhaseRemoveBootstrap: {
 			action(i.initializeKubernetesClients),
+			action(i.addClusterAdminGroup),
```

Comment:
this should allow passing in parameters or be more generic. Because we would need to add more identities.